l2geth/core/vm/evm.go

```diff
@@ -323,6 +326,15 @@ func (evm *EVM) Call(caller ContractRef, addr common.Address, input []byte, gas
 		}
 	}
 
+	// We need to be able to show a status of 1 ("no error") for successful L1 => L2 messages.
+	// Unfortunately, the current way we figure out the success of the transaction (by parsing the
```

Comment:
We should reword this comment to not use words like `Unfortunately` - we should just explain what the solution is. We should also remove the words `temporary hack`